examples/va/nfv/hci-ovs-dpdk-sriov/control-plane/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../../va/nfv/hci-ovs-dpdk-sriov/control-plane-pre-ceph
```

Comment:
Can we have naming ovs-dpdk-sriov-hci similar like tripleo ComputeOvsDpdkSriovHci?

Reply:
These templates are based on hci templates and they have same structure than hci. I would prefer to keep this naming in order to maintain it easily